### ISSUE TITLE
Issue with readonly-mode when using Tagify in SELECT mode

### ISSUE THREAD

**Issue by wurst-hans** [yairEO/tagify#1067]
I'm using latest Tagify (v4.13.2). There is an issue with `setReadonly(true)` when using `SELECT` mode. When using your demo page (i.e. https://codepen.io/vsync/pen/yLONYdx) and adding `tagify.setReadonly(true)` as last statement in JS, the dropdown still opens and allows me to select any value.

Using `tagify.setDisabled(true)` or setting `userInput=false` on initialization would works, but both methods are not an option for my current use case.

I'm still looking for a solution to prevent opening dropdown, when Tagify in `SELECT` mode is set to readonly. It would be sufficient, when there is any method, that allows me to modify Tagify settings on the fly after initialization (so I could set `userInput=false` as workaround for the moment).

For now, I'm using the following event handler to prevent opening dropdown, but that's not a neat way, because a blinking cursor remains inside the Tagify:

```
tagify.on('dropdown:show', function (e) {
    var tagify = e.detail.tagify;
    if (tagify.settings.readonly || tagify.settings.disabled) {
        tagify.dropdown.hide(true);
    }
});
```

**Comment by yairEO** [comment]
Hi, thanks for reporting, I will fix it, but I must know why are you actively calling `setReadonly` and not simply adding a `readonly` attribute to your `input` element? This has been the standard way for ~20 years

**Comment by wurst-hans** [comment]
I was wondering why my problem doesn't occur anymore on your example. Later I saw, that you've released 4.13.3 silently :-D

I've built a web application, where forms have to be enabled and disabled dynamically. Ex. one of the use cases is, that the form is set to readonly when clicking submit, until Ajax request has been processed, to prevent any modification. For that, I'm adding `disabled` attribute to `fieldset`, which has no effect on already initialized Tagify elements unfortunately. So, I'm looking for a way to make a Tagify `readonly` completely, but I don't want to make it `disabled`, because disabled elements won't be sent through `POST` on form submit.

If there's a better approach, please let me know.
